lib/models/user-whitelist.js

```diff
@@ -13,4 +14,28 @@ util.inherits(UserWhitelist, Base);
 
 require('../extend-with-factories')(UserWhitelist);
 
+UserWhitelist.prototype.isInTrial = function () {
+  return moment(this.attrs.trialEnd) > moment().utc();
```

Comment:
I don't think this is a big issue actually, since this is a small timing issue which we won't be dealing w/. We are talking days not milliseconds.